intake/create_missing_unsegmented.py

```diff
@@ -0,0 +1,94 @@
+"""Script to re-create an unsegmented csv file from already segmented file. 
+
+This script re-creates file "unsegmented/[file_name].soclog.csv" from existing
+"segmented/[file_name].soclog.seg.csv" file.
+The weaving scripts assumes the structure "unsegmented/[file_name].soclog.csv" 
+to exist. However for some games, it is not found and therefore, it can be 
+re-created by this script. 
+"""
+
+from __future__ import print_function
+
+import argparse
+import csv
+from glob import glob
+import os
+import subprocess
+import sys
+
+from csvtoglozz import utf8_csv_reader
+
+def create_unsegmented_file(dir_corpus, doc, seg_path=''):
+    """remove empty lines and ampersands and write contents to unsegmented/
+
+    Parameters
+    ----------
+    dir_corpus : string
+        Folder of the corpus
+    doc : string
+        Name of the document/game
+    seg_path : string, optional
+        Path to the segmented file we should use ; This is necessary when
+        there are more than one file under segmented/.
+    """
+    dir_corpus = os.path.abspath(dir_corpus)
+    game_dir = os.path.join(dir_corpus, doc)
+    if not os.path.isdir(game_dir):
+        err_msg = 'Unable to find corpus files {}'.format(game_dir)
+        raise ValueError(err_msg)
+
+    # segmented: in ./segmented
+    seg_dir = os.path.join(game_dir, 'segmented')
+    if seg_path:
+        seg_file = os.path.abspath(seg_path)
+        if not os.path.isfile(seg_file):
+            raise ValueError('Unable to locate segmented file {}'.format(
+                seg_path))
+    else:
+        seg_file = glob(os.path.join(seg_dir,
+                                     doc + '*.soclog.seg.csv'))
+        if len(seg_file) != 1:
+            err_msg = 'Unable to locate segmented file {}'.format(seg_file)
+            raise ValueError(err_msg)
+        seg_file = seg_file[0]
+    # paranoid check
+    if not os.path.isfile(seg_file):
+        err_msg = 'Weird error on locating segmented file {}'.format(
+            seg_file)
+        raise ValueError(err_msg)
+        
+    useg_dir = os.path.join(game_dir, 'unsegmented')
+    if not os.path.isdir(useg_dir):
+        os.mkdir(useg_dir)  
+    
+    useg_file = os.path.join(useg_dir, doc + '.soclog.csv')
+    
+    with open(seg_file, 'rb') as incsvfile:  # bytestring
+        csvreader = utf8_csv_reader(incsvfile, delimiter='\t')
+        outcsv = csv.writer(open(useg_file, "wb"), dialect='excel', delimiter='\t')
```

Comment:
You should also add the parameter `lineterminator='\n'` when setting up the CSV writer. I only recently noticed that the default line terminator for the CSV writer is `\r\n` (Windows end of line), which we prefer to avoid for various reasons (one of them being that this type of EOL looks ugly in diffs).